#Standard header
Header header

#######################
# Debug 250-259
#######################
uint16 register_250
uint16 register_251
uint16 register_252
uint16 register_253
uint16 register_254
uint16 register_255
uint16 register_256
uint16 register_257
uint16 register_258
uint16 register_259
